tools/react-app-template/webpack.config.js

```diff
@@ -0,0 +1,145 @@
+
+                // Prefer ESM versions of packages to enable tree shaking
+                mainFields: ["module", "browser", "main"],
+
+                // Polyfill "process" and "util" for lru-cache, webpack 5 no longer does that automatically
+                // Remove this after IE11 support is dropped and lru-cache can be finally upgraded
+                fallback: {
+                    process: require.resolve("process/browser"),
+                    util: require.resolve("util/"),
+                },
+            },
+            module: {
+                rules: [
+                    // TS source files in case TS is used
+                    {
+                        test: /\.tsx?$/,
+                        use: [
+                            {
+                                loader: "babel-loader",
+                            },
+                            {
+                                loader: "ts-loader",
+                                options: {
+                                    transpileOnly: true,
+                                },
+                            },
+                        ],
+                    },
+                    // JS source files in case JS is used
+                    {
+                        test: /\.jsx?$/,
+                        exclude: /node_modules/,
+                        use: [
+                            {
+                                loader: "babel-loader",
+                                options: {
+                                    presets: ["@babel/preset-react"],
+                                },
+                            },
+                        ],
+                    },
+                    {
+                        test: /\.css$/,
+                        use: ["style-loader", "css-loader"],
+                    },
+                    {
+                        test: /\.(eot|woff|ttf|svg|jpg|jpeg|gif)/,
+                        type: "asset/resource",
+                    },
+                    !isProduction && {
+                        test: /\.js$/,
+                        enforce: "pre",
+                        include: path.resolve(__dirname, "src"),
+                        use: ["source-map-loader"],
+                    },
+                ].filter(Boolean),
+            },
+            plugins: [
+                new CaseSensitivePathsPlugin(),
+                // Provide bogus process.env keys that lru-cache, pseudomap and util packages use unsafely for no reason...
+                new EnvironmentPlugin({
+                    npm_package_name: "",
+                    npm_lifecycle_script: "",
+                    _nodeLRUCacheForceNoSymbol: "",
+                    TEST_PSEUDOMAP: "",
+                    NODE_DEBUG: "",
+                }),
+                new ProvidePlugin({
+                    process: "process/browser",
+                }),
+                new DefinePlugin({
+                    PORT: JSON.stringify(PORT),
```

Comment:
@kandl I did not find `PORT` to be used anywhere in the code base. Is it needed for any dependency code?